shell/initialize/plugins.js

```diff
@@ -0,0 +1,39 @@
+import '@shell/plugins/extend-router';
+import '@shell/plugins/formatters';
+import '@shell/plugins/global-formatters';
+import '@shell/plugins/i18n';
+import '@shell/plugins/js-yaml';
+import '@shell/plugins/portal-vue.js';
+import '@shell/plugins/resize';
+import '@shell/plugins/shortkey';
+import '@shell/plugins/tooltip';
+import '@shell/plugins/trim-whitespace';
+import '@shell/plugins/v-select';
+
+import axios from '../utils/axios.js';
+import axiosShell from '@shell/plugins/axios';
+import backButton from '@shell/plugins/back-button';
+import codeMirror from '@shell/plugins/codemirror-loader';
+import cookieUniversalNuxt from '../utils/cookie-universal-nuxt.js';
+import intNumber from '@shell/plugins/int-number';
```

Comment:
Could you take the chance to replace the import as follows? During the migration I got complained that these files do not have an export.
```
import * as intNumber from '@shell/plugins/int-number';
import * as positiveIntNumber from '@shell/plugins/positive-int-number.js';
```

Reply:
Done